util/string.go

```diff
@@ -158,3 +159,28 @@ func StringArrayToUint32Slice(arr pq.StringArray) ([]uint32, error) {
 	}
 	return uint32Slice, nil
 }
+
+func IsSQLInjection(input string) bool {
+	// define patterns that may indicate SQL injection, especially those with a semicolon followed by common SQL keywords
+	patterns := []string{
+		"(?i).+;.*select", // Matches any string with a semicolon followed by "select"
```

Comment:
;select * from users    ==> can't match
Novel;TheSelectionbyKieraCass  ==> can match but it seems too strict.

Reply:
fixed